src/harness/fake_sas.py

```diff
@@ -263,6 +263,12 @@ def TriggerPpaCreation(self, request, ssl_cert=None, ssl_key=None):
   def TriggerDailyActivitiesImmediately(self):
     pass
 
+	  
+  def QueryPropagationAndAntennaModel(self, request, ssl_cert=None, ssl_key=None):
+    from util import QueryPropagationAntennaModel
+     
+    return QueryPropagationAntennaModel(request)
```

Comment:
See other comment later.
I think that the context should be stored in the self. 
For example the NLCD driver, chould be initialized once and save in the "self.nlcd_driver"